cmd/bee/cmd/cmd.go

```diff
@@ -292,6 +299,12 @@ func (c *command) setAllFlags(cmd *cobra.Command) {
 	cmd.Flags().Int(optionReserveCapacityDoubling, 0, "reserve capacity doubling")
 	cmd.Flags().Bool(optionSkipPostageSnapshot, false, "skip postage snapshot")
 	cmd.Flags().Uint64(optionNameMinimumGasTipCap, 0, "minimum gas tip cap in wei for transactions, 0 means use suggested gas tip cap")
+	cmd.Flags().Bool(optionAutoTLSEnabled, false, "Enable AutoTLS for secure WebSocket connections")
+	cmd.Flags().String(optionAutoTLSPort, "5500", "AutoTLS port")
```

Comment:
As this is actually WSS port, it should be set to the next in line of our ports, probably 1635.